pytest_fixtures/api_fixtures.py

```diff
@@ -108,6 +108,7 @@ def module_domain(module_org, module_location):
     return entities.Domain(location=[module_location], organization=[module_org]).create()
 
 
+@pytest.mark.vlan_networking
```

Comment:
As you stated, marking the fixture directly doesn't carry to the tests, this has to be done with a pytest hook reading test item kwargs.

Reply:
@mshriver Does that mean, We had to create a hook or do we have any already in pytest.

Reply:
We would have to create that hook in robottelo. There is a pytest plugin that applies marks automatically from fixtures, but it doesn't enforce the behavior you have in the hooks. Let's just directly mark tests for the time being.

Reply:
Yeah! Marked all of the tests yday!